### PR TITLE
modified presenter to work with deques instead of the obsolete queues

### PR DESCRIPTION
**Pull request by kobi2187**

### CODE REVIEW

File: yaml/presenter.nim [hunk 1]
```diff
@@ -551,7 +551,13 @@ proc doPresent(s: var YamlStream, target: PresenterTarget,
         if levels.len > 0 and levels[levels.high] in
             [dFlowMapStart, dFlowMapValue]:
           raise newException(YamlPresenterJsonError,
-                             "Cannot have sequence as map key in JSON output!")
+
+
+
+
+
+
```
Comment (flyx) [inline yaml/presenter.nim]: PR looks good, but can you remove all these empty lines?

Comment (kobi2187) [reply yaml/presenter.nim]: ok, did that. (from the online editor)
btw, I'm new to github - btw how do I do that locally?

Comment (kobi2187) [reply yaml/presenter.nim]: @flyx ok now?

**Comment by flyx** [comment]
Good now. Thanks!

**Pull request merged**